### PR TITLE
#23 Add Dependency & Validations

### PR DESCRIPTION
**Pull request by ajohnsson95**
Spring-boot-starter-validation dependency added
Validtaions added in MessageDto & UserDto

### Related Issue(s)
#23 

### Proposed Changes
Suggested changes

1. Add Spring-boot-starter-validation dependency 
2. Changes in Dto Records add validations for the variables that are being used 

### Description
Dependency added to handle new validations in Dto records

### CODE REVIEW

File: src/main/java/dev/ebrydeu/spring_boot_library/domain/dto/MessageDto.java [hunk 1]
```diff
@@ -1,15 +1,26 @@
...
 
 public record MessageDto(
         Long id,
+        @NotBlank(message = "Title must not be blank")
+        @Size(max = 100, message = "Title must not exceed 100 characters")
         String title,
+        @NotBlank(message = "Body must not be blank")
         String body,
+        @NotBlank(message = "Author must not be blank")
         String author,
```
Comment (ValentinaSukonina) [inline src/main/java/dev/ebrydeu/spring_boot_library/domain/dto/MessageDto.java:19]: Should not we remove Author column? I think author is a name of the column that will be shown on web, we would need to map our user by id to it

Comment (ajohnsson95) [reply src/main/java/dev/ebrydeu/spring_boot_library/domain/dto/MessageDto.java:19]: Yes we can probably remove it :)

**Comment by ajohnsson95** [comment]
Removed author from the Dto, belive there is still one in the Message entity that we should remove aswell if we've decided to get rid of it completly

**Comment by ajohnsson95** [comment]
author is back again, need to remove it from tests etc aswell otherwise sonarcloud will fail, can take a look at it tomorrow again to make sure it's being properly removed from all instances

**Comment by ValentinaSukonina** [comment]
> author is back again, need to remove it from tests etc aswell otherwise sonarcloud will fail, can take a look at it tomorrow again to make sure it's being properly removed from all instances

I think I fixed those issues in my pull request. I used your current pull and added changes on top ot it, hope you do not mind.

**Pull request merged**